th/2017/winter.json

```diff
@@ -1,1283 +1,1281 @@
 {
-   "name":"ชื่อ",
-   "group":"แฟนซับ",
-   "message":"ไม่ทราบข้อมูล",
-   "lbl_tv":"TV",
-   "lbl_ova":"OVA/ONA/SP",
-   "lbl_movie":"Movie",
-   "tv":[
-      {
-         "name":"ACCA 13-ku Kansatsu-ka",
-         "image":"https://cdn.anilist.co/img/dir/anime/reg/21823-rO1rsx6OANhJ.jpg",
-         "group":[
-            {
-               "status":"release",
-               "detail":[
-                  {
-                     "name":"Shiniji-FS"
-                  },
-                  {
-                     "name":"VirgoNeo-FS"
-                  }
-               ]
-            },
+  "name": "ชื่อ",
+  "group": "แฟนซับ",
+  "message": "ไม่ทราบข้อมูล",
+  "lbl_tv": "TV",
+  "lbl_ova": "OVA/ONA/SP",
+  "lbl_movie": "Movie",
+  "tv": [
+    {
+      "name": "ACCA 13-ku Kansatsu-ka",
+      "image": "https://cdn.anilist.co/img/dir/anime/reg/21823-rO1rsx6OANhJ.jpg",
```

Comment:
Solve on 651d5d2 

PS. Spirit Pact isn't exist on Myanimelist. so it still use image from Anilist.co